dev/scripts/server545_puppet3_compatibility.sh

```diff
@@ -5,7 +5,7 @@
 
 
 # Puppet Server package version (from http://builds.puppetlabs.lan/puppetserver/)
-export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.04.17T0057'
+export PACKAGE_BUILD_VERSION='2.1.0.SNAPSHOT.2015.05.14T1222'
```

Comment:
Ha yeah this script isn't used by CI at all, not sure exactly why it ended up here in the first place but we can probably delete it once we prove out the CI pipeline.